at-sample/spring-seata/src/main/resources/application.properties

```diff
@@ -2,4 +2,4 @@ spring.application.name=spring-seata
 spring.datasource.driverClassName=com.mysql.cj.jdbc.Driver
 spring.datasource.url=jdbc:mysql://127.0.0.1:3306/seata?useSSL=false&useUnicode=true&characterEncoding=UTF8
 spring.datasource.username=root
-spring.datasource.password=123456
\ No newline at end of file
+spring.datasource.password=
```

Comment:
if we remove hard-code credential, it means we should write it in another place? the application can not read password.

Reply:
What I want to do is put it in the e2e replace file, not in the sample itself.

Reply:
> What I want to do is put it in the e2e replace file, not in the sample itself.

The hard-code password is existed in file before the case be implmented as e2e-case. Dose it mean we do not want any hard-code password exist in config file now ?

Reply:
I think e2e is just one entry point to run the sample, and we first need to make sure that the underlying code and configuration are free of sensitive information. It is possible to exist and replace upper-level runtimes, and it is best to centrally control upper-level sensitive information in workflow.

Reply:
@xjlgod I will close this PR.  Look forward to your solution.